### PR TITLE
TC_07.03.004 | Folder > Configure>check redirect to configure page

### PR DESCRIPTION
**Pull request by JackBlaaack**
https://trello.com/c/w5QiBDPn/338-tc0703004-folder-configurecheck-redirect-to-configure-page

### CODE REVIEW

File: cypress/e2e/redirectToConfigurePage.cy.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+
+import folderName from "../fixtures/createFolder.json";
+import title from "../fixtures/folderConfigure.json"
+
+const PORT = Cypress.env("local.port");
+const HOST = Cypress.env('local.host');
+
+beforeEach('Create folder',() => {
```
Comment (Rom-T) [inline cypress/e2e/redirectToConfigurePage.cy.js:9]: BeforIch add discribe inside. And make a code format.

File: cypress/e2e/redirectToConfigurePage.cy.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+
+import folderName from "../fixtures/createFolder.json";
+import title from "../fixtures/folderConfigure.json"
+
+const PORT = Cypress.env("local.port");
+const HOST = Cypress.env('local.host');
```
Comment (Rom-T) [inline cypress/e2e/redirectToConfigurePage.cy.js:7]: Variables are not needed if you use them only once.

Comment (JackBlaaack) [reply cypress/e2e/redirectToConfigurePage.cy.js:7]: I'll do it, that don't write hardcode in my url.

File: cypress/e2e/redirectToConfigurePage.cy.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('redirect to congigure page', () => {
+    beforeEach('Create folder',() => {
+        cy.get('span').contains('Create a job').click();
+        cy.get('#name').type(folderName.folderName);
+        cy.get('li.com_cloudbees_hudson_plugins_folder_Folder').scrollIntoView();
+        cy.wait(2000);
```
Comment (S-Lanchik) [inline cypress/e2e/redirectToConfigurePage.cy.js:14]: If I correctly understand your test, you don't need lines 13 and 14 here.

File: cypress/e2e/redirectToConfigurePage.cy.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        cy.get('li.com_cloudbees_hudson_plugins_folder_Folder').click();
+        cy.get('#ok-button').click();
+        cy.get('a[href="/job/JenkinsFolder/"]').click();
+    });
+it('test', () => {
```
Comment (S-Lanchik) [inline cypress/e2e/redirectToConfigurePage.cy.js:19]: Write the TC name with number inside 'it'.

File: cypress/e2e/redirectToConfigurePage.cy.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+   cy.get('span').contains('Configure').click();
```
Comment (S-Lanchik) [inline cypress/e2e/redirectToConfigurePage.cy.js:20]: Put 'Configure' to fixtures.

**Pull request closed without merging**